src/pages/code-sample/index.md

```diff
@@ -193,8 +192,7 @@ curl -X POST \
     {
       "href":"<SIGNED_POST_URL>",
       "storage":"<storage>",
-      "type":"vnd.adobe.photoshop",
-      "overwrite":true
```

Comment:
@khound maybe change these lines, in all examples to  either
`"overwrite":<overwrite_if_applicable>` or
`"overwrite":<overwrite>`
to indicate the value is potentially available here.  Thoughts?

Reply:
@noahhoffman For code sample page actually we changed all the examples with storage ==`external` as our docs are targeted for external customers. So I think better not to have `overwrite` flag here.